### PR TITLE
Inject hero images

### PR DESCRIPTION
**Pull request by davidrapson**
Add api method and middleware to allow hero images to be injected from the CMS. Allows hero images to be stored in a single place in the CMS and means hero images are consistently optimised through Imgix. Updates `/funding`, `/research` and `/about` to use images from the CMS.

### CODE REVIEW

File: middleware/inject-hero.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const { defaultHeroImage } = require('../modules/images');
+
+module.exports = function injectHeroImage(page) {
+    const heroSlug = get('heroSlug')(page);
+    return function(req, res, next) {
+        if (heroSlug) {
+            contentApi
+                .getHeroImage({
+                    locale: req.i18n.getLocale(),
+                    slug: heroSlug
```
Comment (mattandrews) [inline middleware/inject-hero.js]: hero-slug

File: middleware/inject-hero.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                })
+                .then(heroImage => {
+                    res.locals.heroImage = heroImage;
+
+                    console.log(heroImage);
```
Comment (mattandrews) [inline middleware/inject-hero.js]: Can remove this.

File: views/pages/toplevel/about.njk [hunk 3]
```diff
@@ -5,7 +5,6 @@
 
 {% block title %}{{ title }} | {% endblock %}
 
-{% set heroImage = heroImages.mentalHealthFoundation %}
```
Comment (mattandrews) [inline views/pages/toplevel/about.njk]: Worth removing this (and the others) from the hero config file (and the images themselves) to avoid confusion?

Comment (davidrapson) [reply views/pages/toplevel/about.njk]: Yep, but I'll come back and do this afterwards as a bit of cleanup. The images themselves can go then too.

**Comment by davidrapson** [comment]
FYI: this will probably fail until the endpoint exists.

**Pull request closed without merging**